src/services/logs/access/types.ts

```diff
@@ -0,0 +1,44 @@
+import { OptionsWithRql } from '../../types';
+
+export interface LogsAccessService {
+  /**
+   * ## Retrieve a list of access logs
+   *
+   * **Global Permissions:**
+   * - `VIEW_ACCESS_LOGS` - Allows a user to view access logs
+   * @param options {@link OptionsWithRql} - Add filters to the requested list
+   * @returns An array of access logs {@link AccessLog AccessLog[]}
+   */
+  find(options?: OptionsWithRql): Promise<AccessLog[]>;
```

Comment:
You're probably thinking to yourself... "Hey Chad, how come this find method isn't using a pager unlike the other find methods in the SDK" and don't worry, I'll tell you.

Whilst the Log Service supports the paging model, it is not implemented on the service side, thus we return just the results queried without any magic.